dts/arm/nxp/nxp_rt1024.dtsi

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2020, Linaro
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <nxp/nxp_rt.dtsi>
+
+&sysclk {
+	clock-frequency = <500000000>;
+};
+
+/* i.MX rt1024 default FlexRAM partition:
+ * ITCM: 64KB
+ * DTCM: 64KB
+ * OCRAM: 128KB
+ */
+&itcm {
+	reg = <0x00000000 0x10000>;
```

Comment:
Please update the ITCM, DTCM, OCRAM, and QSPI sizes to use `DT_SIZE_K` or `DT_SIZE_M`. See #29420

Reply:
Ok, I have modified codes according to new form.